glean-core/python/tests/test_glean.py

```diff
@@ -303,16 +315,19 @@ def test_ping_collection_must_happen_after_currently_scheduled_metrics_recording
         name=ping_name, include_client_id=True, send_if_empty=False, reason_codes=[]
     )
     string_metric = StringMetricType(
-        disabled=False,
-        category="category",
-        lifetime=Lifetime.PING,
-        name="string_metric",
-        send_in_pings=[ping_name],
+        CommonMetricData(
+            disabled=False,
+            category="category",
+            lifetime=Lifetime.PING,
+            name="string_metric",
+            send_in_pings=[ping_name],
+            dynamic_label=None,
+        )
     )
 
     # This test relies on testing mode to be disabled, since we need to prove the
     # real-world async behaviour of this.
-    Dispatcher._testing_mode = False
+    # Dispatcher._testing_mode = False
```

Comment:
Was leaving this commented out intentional?

Reply:
And here I thought I removed them.